curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-034.md

```diff
@@ -1,34 +1,32 @@
 ---
-id: 6193fb8bef24145b57fbed00
+id: 61fdafe6f07fd7a1c6785bc2
 title: Step 34
 challengeType: 0
 dashedName: step-34
 ---
 
 # --description--
 
-Now you'll need to adjust the layout of the dollar values. The CSS `:not()` selector allows you to target all elements matching a given selector *except* those which match the selector you provide within `:not()`.
-
-Create a `span:not(.name)` selector to target all of your `span` elements except those with the `class` set to `name`. Give it a `margin-left` property set to `10px` and a `min-width` property set to `15%`.
+Finally, you need to take these hidden elements out of the document flow. Give the span[class~="sr-only"] ` selector a `position` property set to `absolute`, a `padding` property set to `0`, and a `margin` property set to `-1px`. This will ensure that not only are the no longer visible, but they are not even within the page view.
```

Comment:
```suggestion
Finally, you need to take these hidden elements out of the document flow. Give the `span[class~="sr-only"]` selector a `position` property set to `absolute`, a `padding` property set to `0`, and a `margin` property set to `-1px`. This will ensure that not only are they no longer visible, but they are not even within the page view.
```